base/bitarray.jl

```diff
@@ -1779,26 +1779,41 @@ end
 # map across the chunks. Otherwise, fall-back to the AbstractArray method that
 # iterates bit-by-bit.
 function bit_map!(f::F, dest::BitArray, A::BitArray) where F
-    size(A) == size(dest) || throw(DimensionMismatch("sizes of dest and A must match"))
+    length(A) <= length(dest) || throw(DimensionMismatch("length of dest must be >= than length of A"))
```

Comment:
`>=` is read "greater than or equal to", so it makes more sense to omit "than" from the sentence. Also, the `map!` docstring names the function arguments `destination` and `collection...`. (There is no docstring specific to the `BitArray` method.)

```suggestion
    length(A) <= length(dest) || throw(DimensionMismatch("length of destination must be >= length of collection"))
```